### ISSUE TITLE
Duplicated environment variables

### ISSUE THREAD

**Issue by chrisfarms** [alphagov/paas-cf-conduit#3]
# What

Any existing environment variables are not overridden when setting up the env for executing commands.

# Steps to reproduce

```
$ PGDATABASE=xxx cf conduit posty -- env | grep PGDATABASE
PGDATABASE=xxx
PGDATABASE=rdsbroker_8xxxxx_2xxxxx_bxxx_bxxxxxxxxx
```

# What is expected

Only the single (relevant) PGDATABASE variable in the environment.

# Notes

Overwrite existing environment variables before exec.

**Comment by alext** [comment]
We could fix this by upgrading to Go 1.9 - https://golang.org/doc/go1.9#os/exec

**Comment by 46bit** [comment]
I'm working on this in the https://github.com/alphagov/paas-cf-conduit/tree/test-for-env-mishandling branch, following an approach suggested by @alext https://github.com/alphagov/paas-cf-conduit/pull/9#issuecomment-344201493

**Comment by camelpunch** [comment]
Fixed by #17
